test/test.googleauth.ts

```diff
@@ -1225,6 +1226,20 @@ it('should accept credentials to get a client', async () => {
   assert.equal(client.email, 'hello@youarecool.com');
 });
 
+it('should allow passing scopes to get a client', async () => {
+  const scopes = ['http://examples.com/is/a/scope'];
+  const keyFilename = './test/fixtures/private.json';
+  const client = await auth.getClient({scopes, keyFilename}) as JWT;
+  assert.equal(client.scopes, scopes);
+});
+
+it('should allow passing a scope to get a client', async () => {
```

Comment:
Put the comment in a wrong commit. Trying again.
Better to have a description than the test above.